content/docs/csidriver/installation/operator/unity.md

```diff
@@ -107,12 +107,12 @@ metadata:
   namespace: unity
 spec:
   driver:
-    configVersion: v2.7.0
+    configVersion: v2.8.0
```

Comment:
If my understanding is correct, there would be no release for dell-csi-operator in CSM 1.8. So you could revert the changes in this file completly